File: lib/src/main/java/uk/iatom/rfm69/registers/values/OokFix.java
package uk.iatom.rfm69.registers.values; import uk.iatom.rfm69.registers.IRegisterValue;

public enum OokFix implements IRegisterValue {
    
   
    // RegOokFix),
    FIXEDTHRESH_VALUE(0x06); // Default),;

    private byte val;
    
    OokFix(int val) {
        this.val = (byte) val;
    }

    @Override public byte val() {
        return val;
    }
    
}

File: lib/src/main/java/uk/iatom/rfm69/registers/values/OokFix.kt
package uk.iatom.rfm69.registers.values

import uk.iatom.rfm69.registers.IRegisterValue

enum class OokFix(`val`: Int): IRegisterValue { // RegOokFix),
FIXEDTHRESH_VALUE(0x06);


    // Default),;
    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
